### PR TITLE
AbuseIPDB Connector and Playbooks

### PR DESCRIPTION
**Pull request by averbn**

### CODE REVIEW

**Comment by manishkumar1991** [comment]
Hello SOC prime team ,

I all the readme files ,, provided installation or configuring instruction are not matching or not able to be followed after deploying the playbooks .

I have added the API key as well in, in connection but still it shows red exclamation sign

File: Solutions/AbuseIPDB/Playbooks/AbuseIPDB-BlacklistIpToThreatIntelligence/readme.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+2. Fill in the required paramteres:
+    * Playbook Name: Enter the playbook name here
+
+[![Deploy to Azure](https://aka.ms/deploytoazurebutton)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2FAzure-Sentinel%2Fmaster%2FSolutions%2FAbuseIPDB%2FPlaybooks%2FAbuseIPDB-BlacklistIpToThreatIntelligence%2Fazuredeploy.json) [![Deploy to Azure](https://aka.ms/deploytoazuregovbutton)](https://portal.azure.us/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2FAzure-Sentinel%2Fmaster%2FSolutions%2FAbuseIPDB%2FPlaybooks%2FAbuseIPDB-BlacklistIpToThreatIntelligence%2Fazuredeploy.json)
+
+
+### Post-Deployment instructions 
+#### a. Authorize connections
+Once deployment is complete, authorize each connection.
+1.	Click the AbuseIPDB connection resource
```
Comment (manishkumar1991) [inline Solutions/AbuseIPDB/Playbooks/AbuseIPDB-BlacklistIpToThreatIntelligence/readme.md:21]: More steps need to be added
1> Open playbook which has been deployed 
2> click API connection on left side blade
3>  Click the AbuseIPDB connection resource
4> Click edit API connection
5> Click Authorize
6> Sign in
7> Click Save
8> Repeat steps for tiIndicators connector API Connection.

Comment (averbn) [reply Solutions/AbuseIPDB/Playbooks/AbuseIPDB-BlacklistIpToThreatIntelligence/readme.md]: done

**Comment by averbn** [comment]
> Hello SOC prime team ,
> 
> In all the readme files ,, provided installation or configuring instruction are not matching or not able to be followed after deploying the playbooks .
The problem seems to be that I specified the final links - which will not work until the merge occurs. I changed back the links to the SOCprime clone for your testing.(https://github.com/socprime/Azure-Sentinel/tree/AbuseIPDB/Solutions/AbuseIPDB/Playbooks).  Before merging into the master, notify me so that I change the links to the master.

File: Solutions/AbuseIPDB/Playbooks/AbuseIPDB-BlacklistIpToThreatIntelligence/readme.md (hunk 1 above)
Comment (manishkumar1991) [reply Solutions/AbuseIPDB/Playbooks/AbuseIPDB-BlacklistIpToThreatIntelligence/readme.md]: Change the same in all readme.md files except custom connector readme file .

**Review by manishkumar1991 — DISMISSED** [review]
approved

**Comment by manishkumar1991** [comment]
> The problem seems to be that I specified the final links - which will not work until the merge occurs. I changed back the links to the SOC Prime clone for your testing.(https://github.com/socprime/Azure-Sentinel/tree/AbuseIPDB/Solutions/AbuseIPDB/Playbooks). Before merging into the master, notify me so that I change the links to the master.

@averbn  please reset the links to original .

**Review by manishkumar1991 — APPROVED** [review]
approved

**Comment by manishkumar1991** [comment]
@averbn Please resolve the commit , so that we can go for merge

**Comment by averbn** [comment]
> @averbn Please resolve the commit conflict, so that we can go for merge

It seems some unresolved issues with this PR. So I created new PR with all files: https://github.com/Azure/Azure-Sentinel/pull/5192

**Comment by v-spadarthi** [comment]
@averbn : You have raised a new [PR](https://github.com/Azure/Azure-Sentinel/pull/5192) with same files so,we are closing this PR. Thanks !

**Pull request closed without merging**